shard.yml

```
@@ -4,11 +4,11 @@ version: 0.1.0
 authors:
   - Jeremy Woertink <jeremywoertink@gmail.com>
 
-crystal: ">= 0.35.1"
+crystal: ">= 0.35.1, < 2.0.0"
```

```suggestion
crystal: ">= 0.36.1, < 2.0.0"
```

To match our testing plan